### PR TITLE
Fill in TODOs and update opinions

### PR DESCRIPTION
**Pull request by lcurry**
Filled in some of the TODOs from the tables, and updates to opinions

### CODE REVIEW

File: docs/modules/con-tssc-opinions.adoc [hunk 1]
```diff
@@ -11,13 +11,16 @@ The {ProjectName} {ProjectShortName} is an opinionated software development and
...
+*  The "Release Branch" is the branch that goes into production. The term "main" will be used to refer to the release branch. The name of this branch may be different from team to team.  There are many different naming conventions for a branch that goes to production (e.g. "master", "trunk", "release", etc.) 
 
 *  The minimum, and recommended, number of logical environments is
    - `N` number of development environments, one per active feature branch
    - Test
    - Production
 
+*  For UAT testbed infrastructure (e.g. Selenium Grid), such infrastructure, when possible, should be dynamically created on-demand (by infrastructure-as-code) using TSSC step. There will be cases where UAT infrastructure already exists in-house. In such cased, the use of pre-existing/static infrastructure is a supported alternative for UAT testing.
```
Comment (ppremru) [inline docs/modules/con-tssc-opinions.adoc]: Hmm.   I think the sentence needs tuning.  
- What is meant by 'using TSSC step'?
- Change "In such cased" to "In such cases"

File: docs/modules/ref-tssc-workflow-components.adoc [hunk 2]
```diff
@@ -148,7 +148,7 @@ Detect new, changed, or merged,  Merge Request to Release Branch
 * Application Language Packaging Tool
 
 | Static Code Analysis
-| The pipeline will perform static analysis on source code to catch defects, vulnerabilities, programatic and stylistic problems as early in the development lifecycle as possible, prior to deploying and running the code.
+| The pipeline will perform static analysis on source code to catch defects, vulnerabilities, programmatic and stylistic problems as early in the development life cycle as possible, prior to deploying and running the code.
```
Comment (ppremru) [inline docs/modules/ref-tssc-workflow-components.adoc]: Just an opinion:   Maybe make this two sentences?   (I did not care for  'running the code'...)

The pipeline will perform static analysis on source code to identify defects, vulnerabilities, programmatic and stylistic problems as early in the development life cycle as possible.  For example, static analysis is completed prior to building, scanning and deploying the image.

File: docs/modules/ref-tssc-workflow-components.adoc [hunk 3]
```diff
@@ -255,52 +255,52 @@ Detect new, changed, or merged,  Merge Request to Release Branch
 * Source Control Tool
 
 | Deploy or Update TEST Environment
-| #TODO - purpose#
+| Deploy code to shared test environment with latest feature available for shared testing
```
Comment (ppremru) [inline docs/modules/ref-tssc-workflow-components.adoc]: Suggestion:  
Deploy image built from the latest release branch to the test environment

File: docs/modules/ref-tssc-workflow-components.adoc [hunk 4]
```diff
@@ -255,52 +255,52 @@ Detect new, changed, or merged,  Merge Request to Release Branch
...
 
 | Validate Test Environment Configuration
-| #TODO - purpose#
+| Verify that the test environment has been built successfully and configured according to predefined specifications and rules
 |
 * Validate Environment Configuration Tool
 
 | Run Performance Tests (Full)
-| #TODO - purpose#
+| To determining the speed, responsiveness and stability impact of new development when under workload
```
Comment (ppremru) [inline docs/modules/ref-tssc-workflow-components.adoc]: Suggestion: Execute tests to determine the speed, responsiveness and stability

File: docs/modules/ref-tssc-workflow-components.adoc [hunk 5]
```diff
@@ -255,52 +255,52 @@ Detect new, changed, or merged,  Merge Request to Release Branch
...
 
 | Deploy or Update TEST Environment
-| #TODO - purpose#
+| Deploy code to shared test environment with latest feature available for shared testing
 |
 * Continuous Deployment Tool
 
 | Validate Test Environment Configuration
-| #TODO - purpose#
+| Verify that the test environment has been built successfully and configured according to predefined specifications and rules
```
Comment (ppremru) [inline docs/modules/ref-tssc-workflow-components.adoc]: Suggestion: Using predefined rules, validate the configuration files used to deploy the test environment

**Review by ppremru — COMMENTED** [review]
Added a few suggestions ...

File: docs/modules/con-tssc-opinions.adoc (hunk 1 above)
Comment (lcurry) [reply docs/modules/con-tssc-opinions.adoc]: Fixed spelling.  
Changed term to "step runner" and added definition of step runner to opinions.

File: docs/modules/ref-tssc-workflow-components.adoc (hunk 2 above)
Comment (lcurry) [reply docs/modules/ref-tssc-workflow-components.adoc]: Updates made. Much better thanks.

File: docs/modules/ref-tssc-workflow-components.adoc (hunk 3 above)
Comment (lcurry) [reply docs/modules/ref-tssc-workflow-components.adoc]: Updated as suggested

File: docs/modules/ref-tssc-workflow-components.adoc (hunk 4 above)
Comment (lcurry) [reply docs/modules/ref-tssc-workflow-components.adoc]: Update as suggested

File: docs/modules/ref-tssc-workflow-components.adoc (hunk 5 above)
Comment (lcurry) [reply docs/modules/ref-tssc-workflow-components.adoc]: Update as suggested

**Review by HunterGerlach — APPROVED** [review]
Looks good!

**Review by ppremru — APPROVED** [review]

**Review by dwinchell — APPROVED** [review]

**Pull request merged**